lib/std/os.zig

```diff
@@ -4622,7 +4612,7 @@ pub const UnexpectedError = error{
 
 /// Call this when you made a syscall or something that sets errno
 /// and you get an unexpected error.
-pub fn unexpectedErrno(err: usize) UnexpectedError {
+pub fn unexpectedErrno(err: anytype) UnexpectedError {
```

Comment:
hmm, why do we need this?

Also note that if `unexpected_error_tracing` is false then people might start accidentally passing other types to this (enums/structs) and have no idea until someone turns on unexpected error tracing

Reply:
to cite myself "maybe it could be more precise and accept only .Int type". I added a check on the type.